programs/marginfi/src/ix_utils.rs

```diff
@@ -0,0 +1,262 @@
+use anchor_lang::solana_program::sysvar::instructions::{
+    get_instruction_relative, load_instruction_at_checked,
+};
+use anchor_lang::{prelude::*, solana_program::instruction::Instruction};
+
+use crate::constants::COMPUTE_PROGRAM_KEY;
+use crate::{MarginfiError, MarginfiResult};
+
+/// Structs that implement this trait have a `get_hash` tool that returns the function discriminator
+pub trait Hashable {
+    fn get_hash() -> [u8; 8];
```

Comment:
functionally it's a hash but even here in the comment you call it the discriminator, why call it the hash. Regardless though DISCRIMINATOR exists anyway.